components/scream/cmake/machine-files/compy.cmake

```diff
@@ -0,0 +1,7 @@
+set(CMAKE_CXX_FLAGS "-w" CACHE STRING "")
```

Comment:
Why do we want to ignore all warnings? Also, this is a gcc flag, I'm not sure it will work with Intel (which I presume someone might try to use on compy), so I would leave it out.

Reply:
Thanks . I will remove it. I have been using gcc (with `-Wall` added explicitly with `-D`) but, you are right, it wont work with other compilers. I will fix the ARCH flag as well.